src/main/java/com/fzakaria/mvn2nix/model/MavenNixInformation.java

```diff
@@ -6,18 +6,15 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.TreeMap;
 
 @Immutable
 public class MavenNixInformation {
 
     private final Map<String, MavenArtifact> dependencies;
 
     public MavenNixInformation(Map<String, MavenArtifact> dependencies) {
-        this.dependencies = new HashMap<>(dependencies);
-    }
-
-    public Map<String, MavenArtifact> getDependencies() {
-        return new HashMap<>(dependencies);
+        this.dependencies = new TreeMap<>(dependencies);
```

Comment:
Maybe add a comment explaining why it needs to be a TreeMap?

Did I have any test in this repo?
I can't remember :)